newton/_src/solvers/mujoco/solver_mujoco.py

```
@@ -1377,6 +1385,8 @@ def add_geoms(newton_body_id: int):
                     geom_params["solmix"] = shape_geom_solmix[shape]
                 if shape_geom_gap is not None:
                     geom_params["gap"] = shape_geom_gap[shape]
+                if shape_geom_solref is not None:
+                    geom_params["solref"] = convert_solref(ke, kd, 1.0, 1.0)
```

Ruff (0.14.10)

Undefined name `ke`

(F821)